src/components/course/course-header/deprecated/CourseRunCard.jsx

```diff
@@ -100,13 +100,10 @@ const CourseRunCard = ({
 
   const enrollmentUrl = useCourseEnrollmentUrl({
     enterpriseCustomer,
-    key,
```

Comment:
The `useCourseEnrollmentUrl` no longer uses the now-removed properties, so removing them here as well.